src/components/Properties/PropertySelect.vue

```diff
@@ -91,6 +91,23 @@ export default {
 	},
 
 	computed: {
+		/**
+		 * Store getters filtered and mapped to usable object
+		 * This is the list of addressbooks that are available to write
+		 *
+		 * @return {{id: string, name: string}[]}
+		 */
+		selectableOptions() {
+			return this.options
+				.filter(option => option.readOnly === undefined || !option.readOnly)
```

Comment:
```suggestion
				.filter(option => !option.readOnly)
```

Could be simplified.